toqito/matrix_ops/vectors_to_gram_matrix.py

```diff
@@ -3,7 +3,11 @@
 
 
 def vectors_to_gram_matrix(vectors: list[np.ndarray]) -> np.ndarray:
-    """Given a list of vectors, return the Gram matrix."""
```

Comment:
I think one of my diffs takes care of the more in-depth docstring: 
https://github.com/vprusso/toqito/pull/265